app/services/mlops/model_deployment_service.py

```
@@ -0,0 +1,711 @@
+
+                await self._store_deployment(deployment)
+
+                logger.info(
+                    f"Successfully rolled back deployment {deployment_id} to version {target_version}"
+                )
+                return True
+            else:
+                logger.error(f"Failed to rollback deployment {deployment_id}")
+                return False
+
+        except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)